string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,13 @@
+/* exported isVowel */
+
+function isVowel(char) {
+  var isVowel = false;
+  if (char === 'a' || char === 'A' ||
+      char === 'e' || char === 'E' ||
+      char === 'i' || char === 'I' ||
+      char === 'o' || char === 'O' ||
+      char === 'u' || char === 'U') {
```

Comment:
I bet you could simplify this by converting the character to uppercase or lowercase before the conditional!